example/image-classification/README.md

```diff
@@ -126,6 +126,8 @@ to calculate the accuracy.
 | `imagenet1k-resnet-50`    | 0.7527 | 0.9258 |
 | `imagenet1k-resnet-101`   | 0.7684 | 0.9327 |
 | `imagenet1k-resnet-152`   | 0.7653 | 0.9312 |
+| `imagenet1k-resnext-50`   | 0.7602 | 0.9286 |
+| `imagenet1k-resnext-101`  | 0.7828 | 0.9408 |
```

Comment:
does the reulst is based on using ```score.py ``` or just the result during training? because using score.py may be different with during training test phase result.